service/metadata.yaml

```diff
@@ -121,3 +121,52 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+
+    receiver.produced.size:
+      prefix: otelcol.
+      enabled: false
+      description: Size of items emitted from the receiver.
```

Comment:
I would reference the RFC or mention here directly that this is the size "based on ProtoMarshaler.Sizer()"